nexus/src/app/sagas/instance_start.rs

```diff
@@ -918,6 +958,9 @@ mod test {
         let datastore =
             cptestctx.server.server_context().nexus.datastore().clone();
 
+        // Create uplinks
+        // We only eagerly populate NAT entries on switches that have uplinks configured, so we need to
+        // do some switch configuration before starting the instance in order to test that section of logic
```

Comment:
nit: these lines are over 80 columns (just that rustfmt's option for doing this automatically is unstable)